packages/eslint-plugin-qwik/src/noUseVisibleTask.ts

```diff
@@ -6,25 +6,11 @@ export const noUseVisibleTask: Rule.RuleModule = {
     docs: {
       description: 'Detect useVisibleTask$() functions.',
       recommended: true,
-      url: 'https://qwik.builder.io/docs/guides/best-practices/#avoid-registering-dom-events-in-usevisibletask',
+      url: 'https://qwik.builder.io/docs/guides/best-practices/#use-usevisibletask-as-a-last-resort',
     },
     messages: {
-      noUseVisibleTask: `Qwik tries very hard to defer client code execution for as long as possible.
-This is done to give the end-user the best possible user experience.
-Running code eagerly blocks the main thread, which prevents the user from interacting until the task is finished.
-"useVisibleTask$" is provided as an escape hatch.
-
-When in doubt, instead of "useVisibleTask$()" use:
-- useTask$ -> perform code execution in SSR mode.
-- useOn() -> listen to events on the root element of the current component.
-- useOnWindow() -> listen to events on the window object.
-- useOnDocument() -> listen to events on the document object.
-
-Sometimes it is the only way to achieve the result.
-In that case, add:
-// eslint-disable-next-line qwik/no-use-visible-task
-to the line before "useVisibleTask$" to acknowledge you understand.
-`,
+      noUseVisibleTask:
+        'useVisibleTask$ runs eagerly and blocks the main thread. It should be used as a last resort.',
```

Comment:
I prefer the previous message, it is more explanatory and provides to the user more information. It avoids moving on the documentation and furthermore, being very evident, it creates the desired effect, that of drawing attention. If you have reduced it, it's because it bothers you to always see it in the terminal, you can always disable it globally at your own risk.

Reply:
>  it bothers you to always see it in the terminal, you can always disable it globally at your own risk.

That's kind of the issue isn't it? If people like me find it a bit annoying to have those error messages, then they'll have to disable it, removing the desired effect of drawing attention in teams where there might be new developers that aren't comfortable with eslint, etc.

I'll try to propose a message that doesn't lose much info but is more succint.

Reply:
Thanks ❤️ it looks a good compromise